test/tests/file_destructor.py

```diff
@@ -0,0 +1,13 @@
+try:
```

Comment:
Couple minor things as we wait for the travis-ci build to rerun:
- It's ok to count on the `gc` module being available
- let's use 4 spaces for indenting -- we try to follow [PEP8](https://www.python.org/dev/peps/pep-0008/#id12) for our Python code
- we could try calling `gc.collect()` every ~100 loop iterations to reduce that overhead